chaosblade-exec-plugin/chaosblade-exec-plugin-feign/src/main/java/com/alibaba/chaosblade/exec/plugin/feign/FeignProducerEnhancer.java

```diff
@@ -27,12 +28,16 @@ public EnhancerModel doBeforeAdvice(ClassLoader classLoader, String className, O
         }
         String serviceName = getServiceName(object);
         String template = getTemplate(methodArguments);
+        String url = getUrl(methodArguments);
```

Comment:
obtain the url when the template is empty. do not need to obtain the url in advance